.gitignore

```diff
@@ -1 +1,2 @@
 /nerdctl
+build
```

Comment:
You meant _output?

Reply:
it's for the golangcli-lint. It's just downloaded there.